phpstan.neon.dist

```diff
@@ -3,3 +3,10 @@ parameters:
 	paths:
 		- src
 		- tests/KdybyTests
+	ignoreErrors:
+		-
+			message: '~^Return type \(void\) of method Kdyby\\Events\\SymfonyDispatcher::dispatch\(\) should be compatible with return type \(Symfony\\Component\\EventDispatcher\\Event\) of method Symfony\\Component\\EventDispatcher\\EventDispatcherInterface::dispatch\(\)$~'
```

Comment:
This is real problem in **Kdyby/Console**. Functionality is used there 🐼 

- Pipeline: https://travis-ci.org/Kdyby/Console/builds/514382801?utm_source=github_status&utm_medium=notification
- kdyby/events required in dev: https://github.com/Kdyby/Console/blob/master/composer.json#L32

**Possible solution:**
- fix it in kdyby/events (two implementation of SymfonyDispatcher)
- delete (amend) tests in kdyby/console
- ignore tests in kdyby/console (it fails on Symfony 4 dev-master) and hope it will be fixed in Symfony
- convert to Java :trollface:

Reply:
@kozaktomas What is the problem? I dont see your funcionality, you pointed to some passing pipeline.
How can be ignore error problem in other library?